docs/csharp/programming-guide/generics/benefits-of-generics.md

```diff
@@ -33,7 +33,7 @@ Generics provide the solution to a limitation in earlier versions of the common
 
```

Comment:
Another nit completely unrelated to your PR: I'm wondering if the **Benefits of generics** topic can be removed. It sounds very much like an attempt to "sell" generics that was written for .NET Framework 2.0. Generics are now an accepted part of the language/framework, so this seems largely to be a statement of the obvious.

Reply:
Good suggesetion!  Done.